src/ol/Observable.js

```diff
@@ -75,15 +92,13 @@ class Observable extends EventTarget {
   }
 
   /**
-   * Listen once for a certain type of event.
-   * @param {EventTypes<ChildTypes>} type The event type or array of event types.
-   * @param {function(?): ?} listener The listener function.
-   * @return {import("./events.js").EventsKey|Array<import("./events.js").EventsKey>} Unique key for the listener. If
-   *     called with an array of event types as the first argument, the return
-   *     will be an array of keys.
-   * @api
+   * @param {string|Array<string>} type Type.
+   * @param {function(?): (void|boolean)} listener Listener.
+   * @return {import("./events.js").EventsKey|Array<import("./events.js").EventsKey>} Event key.
+   * @protected
    */
-  once(type, listener) {
+  // the jsdoc api annotation follows below the class declaration.
```

Comment:
Doesn't this comment prevent TypeScript from picking up the signature and the `protected` flag?

Reply:
No, it does seem to count as an empty line (which is also possible). As far as I know jsdoc comments always attach to the next javascript symbol (if they are not standalone).

Reply:
I can remove that comment if wanted.